ios/brave-ios/App/ActionExtension/bs.lproj/InfoPlist.strings

```diff
@@ -0,0 +1,2 @@
+/* Bundle display name */
+"CFBundleDisplayName" = "Buka dalam Brave";
```

Comment:
i could not find them translated in transifex. it seems to fallback to a single language